Makefile

```diff
@@ -229,6 +229,8 @@ helm-chart-manifests: generate
 	# Ensure that what we're about to try to replace actually exists (if it doesn't we want to fail)
 	grep -E $(KUBE_RBAC_PROXY) ./charts/azure-service-operator/templates/generated/*_deployment_*
 	perl -pi -e s,$(KUBE_RBAC_PROXY),"{{ .Values.image.kubeRBACProxy }}",g ./charts/azure-service-operator/templates/generated/*_deployment_*
+	# replace hard coded cert-manager version with templating
+	perl -pi -e s,cert-manager.io/v1alpha2,"{{ .Values.certManagerResourcesAPIVersion }}",g ./charts/azure-service-operator/templates/generated/*cert-manager.io*
```

Comment:
I'm reluctant to expand the dependency on perl (even though this is just for v1) - is there a simpler way?

Reply:
Did this with sed instead.